src/pages/Profile.tsx

```diff
@@ -21,45 +23,176 @@ import { SubscriptionCollection } from "../../generated/definitions/backend/Subs
+import {
+  getServiceReviewStatus,
+  ServiceStatus,
+  ValidService
+} from "../utils/service";
 
 const getMail = (email: string) =>
   email && email !== "" ? atob(email) : undefined;
 
 const SubscriptionService = ({
   service,
-  t
+  t,
+  status,
+  atLeastOneVisible
 }: {
   service: Service;
   t: (key: string) => string;
+  status: string;
+  atLeastOneVisible: boolean;
 }) => {
   return service ? (
     <div>
-      <h5>{service.service_id}</h5>
-      <div>
-        {t("service:name")}: {service.service_name}
-      </div>
-      <div>
-        {t("service:department")}: {service.department_name}
+      <h5>
+        <span className="light-text">{t("service:title")}:</span>{" "}
+        <span className="dark-text">{service.service_id}</span>
+      </h5>
+      <div className="my-3">
+        <span className="light-text">{t("service:name")}:</span>{" "}
+        <span className="light-text">{service.service_name}</span>
       </div>
-      <div>
-        {t("service:organization")}: {service.organization_name}
+      <div className="my-3">
+        <span className="light-text">{t("service:organization")}:</span>{" "}
+        <span className="light-text">{service.organization_name}</span>
       </div>
-      <div>
-        {t("service:organization_fiscal_code")}:{" "}
-        {service.organization_fiscal_code}
+      <div className="my-3">
+        <span className="light-text">
+          {t("service:authorized_recipients")}:
+        </span>{" "}
+        <span className="light-text">{service.authorized_recipients}</span>
       </div>
-      <div>
-        {t("service:authorized_recipients")}: {service.authorized_recipients}
+      <div className="my-3">
+        <span className="light-text">{t("service:authorized_ips")}:</span>{" "}
+        <span className="light-text">{service.authorized_cidrs}</span>
       </div>
-      <div>
-        {t("service:authorized_ips")}: {service.authorized_cidrs}
+      <div className="my-3">
+        <span className="light-text">
+          {t("service:max_allowed_payment_amount")}:
+        </span>{" "}
+        <span className="light-text">
+          {service.max_allowed_payment_amount} {t("service:eurocents")}
+        </span>
       </div>
-      <div>
-        {t("service:max_allowed_payment_amount")}:{" "}
-        {service.max_allowed_payment_amount} {t("service:eurocents")}
+      <div className="status-row">
+        <div className="col-md-8">
+          {status === ServiceStatus.DRAFT ||
+          status === ServiceStatus.NOT_FOUND ? (
+            <div className="service-status">
+              <div>
+                <span className="circle" />
+                <div>
+                  <span className="light-text">
+                    {t("service:state")}:&nbsp;
+                  </span>
+                  <span className="dark-text">
+                    {t("profile:service_draft")}
+                  </span>
+                </div>
+              </div>
+            </div>
```

Comment:
Duplicated code that could be configurable

Reply:
Fix with 3819851

Reply:
See also 580f2ab